site/docs/v1/tech/apis/_two-factor-disable-request-body.adoc

```diff
@@ -0,0 +1,16 @@
+==== Request Body
+
+[.api]
+[field]#applicationId# [type]#[UUID]# [optional]#Optional#::
+The Id of the application. If valid, the email template configured in the Application settings will be used, if present. If not present, the email template configured in the Tenant settings will be used. In either case, the corresponding Application object will be available to the template.
+To learn more about overriding email templates, read our link:/docs/v1/tech/guides/configuring-application-specific-email-templates[Application Specific Email Templates] guide.
+
+[field]#code# [type]#[String]# [required]#Required#::
+A valid Multi-Factor verification code. This value should be provided by the User to verify they are able to produce codes.
++
+This may also be a recovery code. If one is provided, all methods are removed. If, after that, a Multi-Factor method is added, a new set of recovery codes will be generated.
+
+[field]#methodId# [type]#[String]# [required]#Required#::
+The Id of the Multi-Factor method to be disabled.
```

Comment:
can we use `two-factor method` here since we are in the Two Factor API?

Reply:
Okay, so replace all mentions of Mult-Factor with `two-factor`? This would follow the path for the API as well. Once you confirm, I will go ahead and make those changes, if that is what you would prefer. I agree that it would be a better convention to follow.

Reply:
The API is the Multi-Factor/TwoFactor api but the path is `two-factor` so it's definitely confusing on which to choose. `Multi-Factor ` was used by all fields so I stuck with that one.

Reply:
made some edits to use `multi-factor`